### ISSUE TITLE
Wrong documentation (On the web site) for supervised training

### ISSUE THREAD

**Issue by marcandrelarochelle** [facebookresearch/fastText#852]
Hi,

Documentation link: https://fasttext.cc/docs/en/options.html

The documentation states the default for -minn is [3] and -maxn is [6] for supervised training, however when looking deeper into the code C++ code (and python) you will notice that when using Supervised the minn and maxnn is set to 0 instead of using the default 3-6 (Look at the parseArgs function).

Link to the code https://github.com/facebookresearch/fastText/blob/master/src/args.cc

Same happens for the loss and learning rate, they are defaulting to different values than the one written in the documentation.

However for the python version, the supervised default parameters are stated correctly:

Link to the python documentation: https://github.com/facebookresearch/fastText/tree/master/python

Thanks

**Comment by Celebio** [comment]
Hi @marcandrelarochelle ,
Thank you very much for pointing out this issue. We will fix it.

Best regards,
Onur
